jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRolesPresenter.java

```diff
@@ -180,37 +234,40 @@ protected void removeGroupFromRole(final String groupName, final String roleName
 
         void removeAllRoles();
 
-        void addUser(String userName, String roleName, CaseRoleAction... actions);
+        void addOwnerAssignment(String roleName, String user, CaseRolesPresenter.CaseRoleAction... actions);
```

Comment:
Once the case owner is assigned, no actions should be available for that assignment, right?
So what is the purpose of the last argument?